test/load-generator/acme/challenge.go

```diff
@@ -0,0 +1,105 @@
+package acme
+
+import (
+	"errors"
+	"fmt"
+	mrand "math/rand"
+	"strings"
+
+	"github.com/letsencrypt/boulder/core"
+)
+
+// ChallengeStrategy is an interface describing a strategy for picking
+// a challenge from a given authorization.
+type ChallengeStrategy interface {
+	PickChallenge(*core.Authorization) (*core.Challenge, error)
+}
+
+const (
+	// RandomChallengeStrategy is the name for a random challenge selection
+	// strategy that will choose one of the authorization's challenges at random.
+	RandomChallengeStrategy = "RANDOM"
+	// HTTP01ChallengeStrategy is the name for a challenge selection strategy that
```

Comment:
I think these three comments can be condensed into "The following challenge strategies will always pick the named challenge type." That's faster to read and less likely to go out of date.

Reply:
Makes sense. Implemented in 73aaf4c, thanks!